transport/src/test/java/io/netty/channel/pool/SimpleChannelPoolTest.java

```diff
@@ -353,4 +356,50 @@ protected void initChannel(LocalChannel ch) throws Exception {
         pool.close();
         group.shutdownGracefully();
     }
+
+    @Test
+    public void testChannelAcquiredException() throws InterruptedException {
+        final LocalAddress addr = new LocalAddress(getLocalAddrId());
+        final EventLoopGroup group = new DefaultEventLoopGroup();
+
+        // Start server
+        final ServerBootstrap sb = new ServerBootstrap()
+              .group(group)
+              .channel(LocalServerChannel.class)
+              .childHandler(new ChannelInitializer<LocalChannel>() {
+                  @Override
+                  protected void initChannel(LocalChannel ch) throws Exception {
+                      ch.pipeline().addLast(new ChannelInboundHandlerAdapter());
+                  }
+              });
+        final Channel sc = sb.bind(addr).syncUninterruptibly().channel();
+
+        // Create pool, acquire and return channels
+        final Bootstrap bootstrap = new Bootstrap()
+              .channel(LocalChannel.class).group(group).remoteAddress(addr);
+        final SimpleChannelPool pool = new SimpleChannelPool(bootstrap, new ChannelPoolHandler() {
+            @Override
+            public void channelReleased(Channel ch) throws Exception {
+            }
+            @Override
+            public void channelAcquired(Channel ch) throws Exception {
+                throw new NullPointerException();
+            }
+            @Override
+            public void channelCreated(Channel ch) throws Exception {
+            }
+        });
+
+        final CountDownLatch latch = new CountDownLatch(1);
```

Comment:
To make this code simpler, you can avoid the `latch` and just use `future.get()`

Reply:
Done.